src/code/z_actor.c

```diff
@@ -407,7 +419,7 @@ void func_8002C124(TargetContext* targetCtx, PlayState* play) {
 
     actor = targetCtx->unk_94;
     if ((actor != NULL) && !(actor->flags & ACTOR_FLAG_27)) {
-        NaviColor* naviColor = &sNaviColorList[actor->category];
+        TargetColor* naviColor = &sTargetColorList[actor->category];
```

Comment:
`TargetColor* targetColor`?

Reply:
will be fixed in later parts, I am limiting touching the functions out of scope from the ones i mentioned in the description